go.mod

```diff
@@ -80,12 +79,12 @@ require (
 	github.com/modern-go/concurrent v0.0.0-20180306012644-bacd9c7ef1dd // indirect
 	github.com/modern-go/reflect2 v1.0.2 // indirect
 	github.com/munnerz/goautoneg v0.0.0-20191010083416-a7dc8b61c822 // indirect
-	github.com/openshift/hypershift v0.0.0-20220525174911-c7c2b57c98ca
+	github.com/openshift/hypershift v0.1.2
```

Comment:
I am a bit confused here, would it make sense to put this in the first require section or in the replace section?

Reply:
Good spot!
I don't know why this moved to the second require, shouldn't only the first one have direct dependencies?
It looks like it happened here: https://github.com/openshift/cluster-network-operator/commit/f58b1a997381731e45801917c4e6d7fe1e5c48ff
If we decide to change that I would prefer to do it in master first, wdyt?

Reply:
https://github.com/openshift/cluster-network-operator/pull/1459#discussion_r882638796 maybe it was just a mistake